### PR TITLE
Fixes to episode 7

### PR DESCRIPTION
**Pull request by mallewellyn**
Issues with in dendrogram/numerical inspections in current version from my re-ordering. 

Have re-written parts of the numerical visualisation section to use the methylation data (currently assumes sample data), so some of the results and narrative were inconsistent.

### CODE REVIEW

File: _episodes_rmd/07-hierarchical.Rmd [hunk 1]
```diff
@@ -224,7 +224,7 @@ with smallest dissimilarity value are then fused.
 
 # Computing a dendrogram
 
-Dendograms are useful tools that plot the grouping of points and clusters into bigger clusters.
+Denrdograms are useful tools that plot the grouping of points and clusters into bigger clusters.
```
Comment (alanocallaghan) [inline _episodes_rmd/07-hierarchical.Rmd:227]: typo

Comment (mallewellyn) [reply _episodes_rmd/07-hierarchical.Rmd]: [9882a1c](https://github.com/carpentries-incubator/high-dimensional-stats-r/pull/174/commits/9882a1ccbb78d309296882c197fd2b4aac55019f)

**Review by ailithewing — APPROVED** [review]

**Comment by ailithewing** [comment]
@mallewellyn Can you check the conflict? I'm not sure whether the plot code needs adapted to using the methyl data?

**Comment by mallewellyn** [comment]
Done but for some reason, cres.rds has disappeared from gh-pages. Fix in commits below!

**Comment by mallewellyn** [comment]
Also #176

**Pull request merged**